service-locator-dns/src/main/resources/reference.conf

```diff
@@ -28,13 +28,21 @@ service-locator-dns {
   # You can of course have multiple translators though and statically declare the translations as
   # your service's configuration (you'll be supplying environment specific configuration quite typically
   # anyway...), and thus keep your service names nice and clean.
+  #
+  # By default though, we don't translate much i.e. we let it all pass through and put the onus on the
+  # client to specify the right service name.
   name-translators = [
     {
-      "^.*$" = "_$0._tcp.marathon.mesos"
+      "^.*$" = "$0"
     }
   ]
   name-translators = ${?SERVICE_LOCATOR_DNS_NAME_TRANSLATORS}
 
-  # The amount of time to wait for a DNS resolution to occur
-  resolve-timeout = 5 seconds
+  # The amount of time to wait for a DNS resolution to occur for the first and second lookups of a given
+  # name.
+  resolve-timeout1 = 1 second
+
+  # The amount of time to wait for a DNS resolution to occur for the third lookup of a given
+  # name.
+  resolve-timeout2 = 2 seconds
```

Comment:
Would it make sense to have something like a backoff plan:
`resolve-timeouts = [1 second, 1 second, 2 seconds, 5 seconds]`

So the user can specify an array of durations.